tests/command_line/conftest.py

```diff
@@ -9,7 +9,7 @@
 from click.testing import CliRunner
 from requests_mock import ANY
 
-from semantic_release.cli import config as CliConfigModule
+from semantic_release.cli import config as CliConfigModule  # noqa: N812
```

Comment:
Wasn't sure if this needs to be pascal case here, or if I we can just do `cliconfigmodule` or `cli_config_module` and update the couple of places where it's used?